library/custom_template/custom_template.php

```diff
@@ -314,10 +314,12 @@ function sortableCallback(elem){
             edit(<?php echo js_escape($type); ?>, <?php echo js_escape($cc_flag); ?>);
         });
         <?php if ($allowTemplateWarning && !$isNN) { ?>
-        let isPromise = top.jsFetchGlobals('custom_template');
-        isPromise.then(msg => {
-            alertMsg(msg.custom_template.templatesWarn, 9000, 'danger', '', 'disable_template_warning');
-        });
+        // teeheehee
+        let msg = xl("These templates are text only and will not render any other formatting other than pure text.") + " ";
```

Comment:
a win for the greatest js function ever... :drum: :drum: :drum: 
`xl`

![2DV](https://user-images.githubusercontent.com/278968/84000291-411e4200-a919-11ea-92f9-a7338f67020e.gif)

Reply:
I get a kick every time I use that function. And of course, if it pleases our cheering section then all the better....